Sources/Identity/CustomerInfoManager.swift

```diff
@@ -35,8 +35,6 @@ class CustomerInfoManager {
     func fetchAndCacheCustomerInfo(appUserID: String,
                                    isAppBackgrounded: Bool,
                                    completion: ((Result<CustomerInfo, BackendError>) -> Void)?) {
-        self.deviceCache.setCacheTimestampToNowToPreventConcurrentCustomerInfoUpdates(appUserID: appUserID)
-
```

Comment:
just to clarify, it _still_ gets set to `now` after the backend call is successful and the cache is set, right?

Reply:
Yeah and there are tests for that too.